scripts/ingests/utils.py

```diff
@@ -194,26 +184,27 @@ def add_names(db, sources=None, other_names=None, names_table=None, verbose=True
     names_table
         table with source and other_names.
         Expecting source name to be first column and other_names in the 2nd.
-    verbose
     """
-    verbosed = VerbosePrint(verbose)
 
     if names_table is not None and sources is not None:
-        raise RuntimeError("Both names table and sources list provided. Provide one or the other")
+        logging.critical("Both names table and sources list provided. Provide one or the other")
+        raise RuntimeError()
 
     names_data = []
 
     if sources is not None or other_names is not None:
         # Length of sources and other_names list should be equal
         if len(sources) != len(other_names):
-            raise RuntimeError("Length of sources and other_names should be equal")
+            logging.critical("Length of sources and other_names should be equal")
+            raise RuntimeError()
 
         for source, other_name in zip(sources, other_names):
             names_data.append({'source': source, 'other_name': other_name})
 
     if names_table is not None:
         if len(names_table[0]) != 2:
-            raise RuntimeError("Each row should have two elements")
+            logging.critical("Each row should have two elements")
+            raise RuntimeError()
```

Comment:
raise RuntimeError should still include the same text. So maybe create a variable for the text, `msg=Each row...` and then pass that to logging.error(msg) and raise RuntimeError(msg).
Also: don't use logging.critical, stick to debug/info/warning/error; critical should be very rare

Reply:
Yeah I wasn't sure on the critical ones; I just assumed to put them in when something is about to cause the code to crash (e.g. an error about to be raised)?

Reply:
Personally, I actually use critical for things I always want to show up, regardless of whether it's an error or not, but that's probably not the best practice.